src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsStreamingTest.java

```diff
@@ -145,8 +146,7 @@ public void testClobsVarcharASCII() throws SQLException, IOException {
                         Clob c = rs.getClob(2);
                         assertEquals(c.length(), lob_data.get(index).length());
                         try (InputStream is = c.getAsciiStream();
-                                Scanner s = new Scanner(is, java.nio.charset.StandardCharsets.US_ASCII)
-                                        .useDelimiter("\\A")) {
+                                Scanner s = new Scanner(is, "US-ASCII").useDelimiter("\\A")) {

```

Comment:
There should be an unclosed resource warning here, but since I wrote this test I can address it in the LOBs PR if necessary.